openapi/multi-yaml/paths/disputes@{id}@response.yaml

```diff
@@ -0,0 +1,165 @@
+patch:
+  summary: Update dispute response
+  description: Updates the dispute response
+  operationId: UpdateDisputeResponse
+  parameters:
+    - $ref: ../components/parameters/id-path.yaml
+    - $ref: ../components/parameters/authorization-header.yaml
+  tags:
+    - Disputes
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          properties:
+            additional_statement:
+              type: string
+              description: any additional evidence or statements
+            cancelation_policy_disclosure:
+              type: string
+              description: an explanation of how and when the customer was shown your refund policy prior to purchase
```

Comment:
Should this also refer to cancelation instead of refund?

Reply:
Same as the other one
https://docs.stripe.com/api/disputes/object#dispute_object-evidence-cancellation_policy_disclosure